docs/snippets/ov_model_snippets.cpp

```diff
@@ -108,7 +108,8 @@ void visualize_example(const std::shared_ptr<ov::Model>& m) {
 // ! [ov:visualize]
 
 void model_inputs() {
-std::shared_ptr<ov::Model> model;
+ov::Core core;
+std::shared_ptr<ov::Model> model = core.read_model("model.xml");;
```

Comment:
Ubuntu 22.04 gcc will check Model is used unintialized, so build fail. Add initialization for Model.